app/utils.py

```diff
@@ -82,3 +85,44 @@ def fetch_cases_by_date(case_date_from, case_date_to):
                     'signed_date': contract['ActivatedDate'],
                 })
     return result
+
+
+def generate_presto_query(event_id=None, from_date=None, to_date=None, currency='BRL'):
```

Comment:
Great job here adding currency, just keep in mind that if PM and stakeholders agree, you can add the currency as a new form field, because as I can see, this attribute is not being used

Reply:
That's right, I wanted to make sure the query supports different currencies but I didn't add an html input to provide it. I will try to add it now. :)